tools/mtouch/Application.cs

```diff
@@ -862,6 +862,23 @@ void Initialize ()
 
 			Namespaces.Initialize ();
 
+			var hasBitcodeCapableRuntime = false;
```

Comment:
This shouldn't be here, you're reintroducing code unrelated code (that was removed recently).